### PR TITLE
docs: update security guide

### PR DESCRIPTION
**Pull request by josephperrott**
Update the security guide to our new vulnerability intake process.

### CODE REVIEW

**Review by MarkTechson — APPROVED** [review]
LGTM

File: aio/content/guide/security.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ You can run the <live-example></live-example> in Stackblitz and download the cod
 
 <header>Reporting vulnerabilities</header>
 
-To report vulnerabilities in Angular itself, email us at [security@angular.io](mailto:security@angular.io).
+Angular is part of Google [Open Source Software Vulnerability Reward Program](https://bughunters.google.com/about/rules/6521337925468160/google-open-source-software-vulnerability-reward-program-rules), for vulnerabilities in Angular please submit your report [here](https://bughunters.google.com/report)
```
Comment (MarkTechson) [inline aio/content/guide/security.md:16]:
```suggestion
Angular is part of Google [Open Source Software Vulnerability Reward Program](https://bughunters.google.com/about/rules/6521337925468160/google-open-source-software-vulnerability-reward-program-rules), for vulnerabilities in Angular please submit your report [here](https://bughunters.google.com/report).
```

**Review by MarkTechson — APPROVED** [review]
LGTM but fix that young period

**Comment by josephperrott** [comment]
This PR was merged into the repository by commit 37c2f3c2ea7180ded324aba658720f8312906a8f.

**Pull request closed without merging**